### PR TITLE
Adds hydroponics shrubs, allowing botanists to grow hedges.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

Alright I'll make this quick:
Replaces one of the rainbow seeds in the exotic seeds crate with a pack of shrub seeds.
Adds a new, growable seed species for shrubs. Shrubs, when planted (similar to kudzu!) plants a solid, weak barrier in hedges.
These hedges block vision, unless trimmed. Thankfully, we already have a hedge trimming skillchip, so using a sharp implement on the hedge will make it non-opaque. 

## Why It's Good For The Game

Not only do we have hedge trimming as a meme skillchip, but we also have hedges. And what's more, hedge sprites look super good and don't deserve to live in ruin fluff hell. This also gives a nice, if rare plant implement for fancy looking decor. 

## Changelog
:cl:
add: Adds shrub plants and seeds, which can be obtained from the exotic seeds crate in cargo. Plant grown shrubs in-hand to plant a full hedge where you're standing.
tweak: The shrub seeds have replaced one of the rainbow bunch seeds in the exotic seeds crate.
tweak: Hedges that block sight can be trimmed with a sharp implement if you have the hedge trimming skillchip.
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
they dont look like squares like irl hedges

**Comment by Losenis** [comment]
Will botanists start with the chip like how engis start with theirs?

**Comment by trexter555** [comment]
will we be able to do topiary like how we can already do sculpting

File: code/modules/hydroponics/grown/misc.dm [hunk 1]
```diff
@@ -261,3 +261,35 @@
...
+	name = "shrub"
+	desc = "A shrubbery, it looks nice and it was only a few credits too. Plant it on the ground to grow a hedge, shrubbing skills not required."
+	icon_state = "shrub"
+
+/obj/item/grown/shrub/attack_self(mob/user)
+	var/turf/player_turf = get_turf(user)
+	if(player_turf?.is_blocked_turf(TRUE))
+		return FALSE
+	user.visible_message("<span class='danger'>[user] begins to plant \the [src]...</span>")
+	if(do_after(user, 58SECONDS, target = user.drop_location(), progress = TRUE))
```
Comment (LemonInTheDark) [inline code/modules/hydroponics/grown/misc.dm:292]:
```suggestion
	if(do_after(user, 58 SECONDS, target = user.drop_location(), progress = TRUE))
```

**Pull request merged**